blockindex/contractstaking/cache.go

```diff
@@ -436,6 +436,9 @@ func (s *contractStakingCache) putHeight(height uint64) {
 }
 
 func (s *contractStakingCache) mergeDelta(delta *contractStakingDelta) error {
+	if delta == nil {
```

Comment:
it panics in L442 when delta is `nil`

Reply:
this function always return nil if do not check input

Reply:
ok, good to know